recipes/lsof/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = 'lsof' %}
+{% set version = "4.89" %}
+{% set sha256 = "17688b122b9e0330042625ae6c15c4486699a11e2483e3d0e97ec1642b93d7b2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url:
+    - http://cdn-fastly.deb.debian.org/debian/pool/main/l/lsof/{{ name }}_{{ version }}+dfsg.orig.tar.gz
```

Comment:
Any reason not to use the debian version with patches or apply the patches?

Reply:
I was not sure about the format (`.tar.xz`) but it seems OK (example: https://github.com/conda-forge/gdb-feedstock/blob/master/recipe/meta.yaml#L11). I will update the URL.